src/routes/cluster_workload_info.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python3
+"""Contains the handlers and business logic for the workload API."""
+from src.schemas.cluster_workload_info_schema import AccountsWorkloadRequestSchema
+
+
+def get_workload_info(body):
+    """Another stub to ensure connexion does not error out."""
+    print(body)
+    accounts = AccountsWorkloadRequestSchema().load(body)
```

Comment:
From where restriction to API Like no of max account/records count etc gets apply ?